relay-server/src/actors/outcome.rs

```diff
@@ -9,24 +9,54 @@ use std::sync::Arc;
 use std::time::Instant;
 
 use actix::prelude::*;
+use actix_web::http::Method;
 use chrono::SecondsFormat;
+use futures::future::Future;
 use serde::{Deserialize, Serialize};
 
-use relay_common::ProjectId;
+use relay_common::{LogError, ProjectId};
 use relay_config::Config;
 use relay_filter::FilterStatKey;
 use relay_general::protocol::EventId;
 use relay_quotas::{ReasonCode, Scoping};
 
+use crate::actors::upstream::SendQuery;
+use crate::actors::upstream::{UpstreamQuery, UpstreamRelay};
 use crate::ServerError;
 
-// Choose the outcome module implementation (either the real one or the fake, no-op one).
-// Real outcome implementation
+// Choose the outcome module implementation (either processing or non-processing).
+// Processing outcome implementation
 #[cfg(feature = "processing")]
-pub use self::kafka::*;
-// No-op outcome implementation
+pub use self::processing::*;
```

Comment:
_nit_: Should we rather name this `kafka` and `http`, since those are the actual implementations? Processing or not is just a setting that controls which implementation is used.

**Edit**: Now I understand the naming. See main review comment.

Reply:
Kafka was the original name, but now it also sends http messages. If we also change the implementation as per your first recommendation (i.e. have an http actor) that the change would make sense